service/controller/cluster_resource_set.go

```diff
@@ -387,6 +388,7 @@ func newClusterResourceSet(config clusterResourceSetConfig) (*controller.Resourc
 				Private: config.APIWhitelist.Private,
 				Public:  config.APIWhitelist.Public,
 			},
+			CIDRBlockAWSCNI:    fmt.Sprintf("%s/%d", config.CalicoSubnet, config.CalicoCIDR),
```

Comment:
might be worth considering this as two params: CNISubnet and CNICIDR, then we aren't tying the config to calico or aws, also we could move the formatting to the template (see below) ?